test/helper.js

```diff
@@ -1,28 +1,30 @@
 'use strict'
 
-var writeStream = require('flush-write-stream')
-var split = require('split2')
-var os = require('os')
-var pid = process.pid
-var hostname = os.hostname()
+const os = require('os')
+const writer = require('flush-write-stream')
+const split = require('split2')
+const pid = process.pid
+const hostname = os.hostname()
+const v = 1
+
+function once (emitter, name) {
+  return new Promise((resolve) => emitter.once(name, resolve))
+}
```

Comment:
Can you please have it listen also to ‘error’ until ‘data’ is emitted?

Reply:
👍